docs/tutorial_2getting_started.rst

```diff
@@ -3,6 +3,8 @@ Getting Started
 .. toctree::
   :hidden:
 
+**Make sure to follow :doc:`Installation <tutorial_1installation>` and install SAM before continuing here!**
```

Comment:
FYI - this link isn't rendering correctly in the built documentation on Read the Docs.